### PR TITLE
speex

### PR DESCRIPTION
**Pull request by juha-h**
added also speex (optional)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -124,6 +144,40 @@ openssl:
 		./Configure android-armv7 && \
 		ANDROID_DEV=$(SYSROOT)/usr make build_libs
 
+.PHONY: opus
+opus:
+	cd opus && \
+		rm -rf include_opus && \
+		CC="arm-linux-androideabi-gcc --sysroot $(SYSROOT)" \
```
Comment (alfredh) [inline Makefile]: using arm-linux-androideabi-gcc assumes that the compiler is in PATH.

can we use the full path instead:

```
CC="$(CC) --sysroot $(SYSROOT)"
```

**Comment by alfredh** [comment]
after zrtp was compiled successfully, I tried to compile baresip which fails:

```
$ make baresip

make[1]: Entering directory '/home/alfredh/git/baresip-android/baresip'
  SH      src/static.c
  CC      build-arm/src/static.o
  CC [m]  build-arm/modules/zrtp/zrtp.o
In file included from /home/alfredh/git/baresip-android/libzrtp/include/zrtp.h:19:0,
                 from modules/zrtp/zrtp.c:8:
/home/alfredh/git/baresip-android/libzrtp/include/zrtp_base.h:52:18: fatal error: sha2.h: No such file or directory
 #include "sha2.h"
                  ^
compilation terminated.
mk/mod.mk:79: recipe for target 'build-arm/modules/zrtp/zrtp.o' failed
make[1]: *** [build-arm/modules/zrtp/zrtp.o] Error 1
make[1]: Leaving directory '/home/alfredh/git/baresip-android/baresip'
Makefile:109: recipe for target 'baresip' failed
make: *** [baresip] Error 2



```

**Comment by juha-h** [comment]
Alfred E. Heggestad writes:

> can we use the full path instead:
> 
> ```
> CC="$(CC) --sysroot $(SYSROOT)"

I made that change in https://github.com/juha-h/baresip-android.

-- juha

**Comment by juha-h** [comment]
Alfred E. Heggestad writes:

> after zrtp was compiled successfully, I tried to compile baresip which
> fails:

i just built everything (opus, zrtp, speex, baresip) using this Makefile
(in my fork):

https://github.com/juha-h/baresip-android/blob/master/Makefile

and baresip was produced without problems.

i didn't need to add libzrtp sub-projects to LFAGS.

-- juha

**Pull request merged**